README.md

```diff
@@ -259,5 +260,6 @@ Questions can be asked by raising an `Issue`.
 - [Victory Chiamaka Wekwa](https://github.com/VictoryWekwa)
 - [Vinicius Ramos](https://github.com/redrookie)
 - [Vivek Singh](https://github.com/vivmost)
+- [Yashasvi Misra](https://github.com/yashasvimisra2798)
```

Comment:
Hey, @hasthamalp, could you kindly remove the extra added name?
```suggestion
```

Reply:
yes
@Vyvy-vi

Reply:
I can still see two names

Reply:
I can still see the name, maybe remove line 263?